src/mame/drivers/prestige.cpp

```diff
@@ -848,6 +850,15 @@ void prestige_state::gl7007sl(machine_config &config)
 	SOFTWARE_LIST(config, "misterx_cart").set_compatible("misterx");
 }
 
+void prestige_state::gl1510sl(machine_config &config)
+{
+	prestige_base(config);
+
+	SOFTWARE_LIST(config, "cart_list").set_original("gl6000sl");
+	SOFTWARE_LIST(config, "misterx_cart").set_compatible("misterx");
+	SOFTWARE_LIST(config, "gl2000_cart").set_compatible("gl2000");
+}
+
```

Comment:
This isn't declared, which will cause compilation errors. Since it also isn't used, it would probably best be removed.